File: IT/Variabili, operatori, strutture di controllo/Esercizi/Java/Esercizio15.java
/* file Esercizio15.java */

public class Esercizio15 {
	public static void main(String [] args) {
		final int M = 10;
		final int N = 10;

		int i = 1;

		while (i <= M) {
			int j = 1;
			while (j <= N) {
				System.out.print((i * j) + "\t");
				j++;
			}
			i++;
			System.out.print(System.lineSeparator());
		}
	}
}

File: IT/Variabili, operatori, strutture di controllo/Esercizi/C++/esercizio15.cpp
/* file esercizio15.cpp */

#include <iostream>

using std::cout;
using std::endl;

int main()
{
	const int M = 10;
	const int N = 10;

	int i = 1;

	while (i <= M) {
		int j = 1;
		while (j <= N) {
			cout << (i * j) << "\t";
			j++;
		}
		i++;
		cout << endl;
	}
}

